EP-Scenario.py

```diff
@@ -0,0 +1,323 @@
+import subprocess as sp
+from time import sleep
+import os
+from zipfile import ZipFile
+import shutil
+import sys
+import ctypes
+from colorama import init as colorama_init
+from colorama import Fore
+from colorama import Style
+import webbrowser
+
+
+colorama_init() # Initialize Colors
+
+
+def is_admin(): # Necessary Check to see if user running is NT/AUTHORITY || Administrator
+    '''Returns True if script is running with administrator privileges'''
+
+    try:
+        return ctypes.windll.shell32.IsUserAnAdmin()
+    except:
+        return False
+
+def run_as_admin():

```

Comment:
Code duplication is bad. Create a file for main functionality and keep the classes AD-Scenario and EP-Scendario in seperate file (as it is now)

Reply:
This is the biggest flaw I would fix. Try to search for code duplications